packages/components/src/range-control/styles/range-control-styles.ts

```diff
@@ -207,6 +201,11 @@ export const ThumbWrapper = styled.span`
 	user-select: none;
 	width: ${ thumbSize }px;
 	border-radius: ${ CONFIG.radiusRound };
+	z-index: 3;
+
+	@media not ( prefers-reduced-motion ) {
+		transition: left ease 0.1s;
+	}
```

Comment:
This animation introduces a subtle delay when dragging the thumb that can make the UI feel unresponsive. We could consider removing it for the time being, and focus this PR on updating the marks design.

(also, in the block editor "custom" version, this animation was only applied when the is-marked class is applied — just noting in case that was deliberate)

Reply:
The animation already exists in padding/margin so removing it could be seen as a regression in the behavior of the component.

Reply:
@jasmussen should we remove the animation or keep it?

Reply:
Here's the animation, though in low framerate GIF:

![with animation](https://github.com/user-attachments/assets/6cc784be-1847-4936-a86a-140bc8b8dff8)

Note the slight disconnect with the cursor.

Here's with no animation:

![no animation](https://github.com/user-attachments/assets/a9852376-60d8-4f46-b7e8-38252a2580f0)

It's not a huge difference, though I tend to think _with_ animation is preferred, as it make it "snap" into place a bit more.

Reply:
Got it — I will then propose that we keep the animation only when the component is showing marks (like it was in the deleted code from `packages/block-editor/src/components/spacing-sizes-control/style.scss`). I can open a PR for that